### DISCUSSION TITLE
Please give me example for row actions

### DISCUSSION THREAD

**Discussion by stojankukrika** [rappasoft/laravel-livewire-tables#384] [Q&A]
Hi,
Can someone give me some examples of how to add actions in each row: edit, show, and delete? No problem if that needs to go to some route and edit do in the blade, only I do not see that in documentation is that possible and how to do that.

**Comment by tuluong-vl** [comment]
https://github.com/rappasoft/laravel-livewire-tables/wiki/Custom-Row-View check here

**Comment by rappasoft** [comment]
https://github.com/rappasoft/laravel-livewire-tables/wiki/Using-the-built-in-cell-formatting#you-could-also-return-a-view

**Comment by Todomed** [comment]
The link calls me to the main page of the package, will there be another link? I also want to perform said actions

**Comment by rappasoft** [comment]
https://rappasoft.com/docs/laravel-livewire-tables/v1/rows/custom-row-view
or
https://rappasoft.com/docs/laravel-livewire-tables/v1/columns/using-the-built-in-cell-formatting
